security/container_security/security-hosts-vms.adoc

```diff
@@ -28,7 +28,7 @@ ifndef::openshift-origin[]
 endif::[]
 * xref:../../installing/install_config/installing-customizing.adoc#installation-special-config-encrypt-disk_installing-customizing[Disk encryption]
 * xref:../../installing/install_config/installing-customizing.adoc#installation-special-config-chrony_installing-customizing[Chrony time service]
-* xref:../../updating/updating-cluster-between-minor.adoc#update-service-overview_updating-cluster-between-minor[{product-title} cluster updates]
+* xref:../../updating/updating-cluster-within-minor.adoc#update-service-overview_updating-cluster-within-minor[{product-title} cluster updates]
```

Comment:
Replace link with:updating/understanding-the-update-service.html#update-service-overview_understanding-the-update-service